DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -408,4 +408,27 @@ public function fixTemplates(ContainerBuilder $container, Definition $definition
 
         $definition->addMethodCall('setTemplates', array($definedTemplates));
     }
+
+    /**
+     * Replace the empty arguments required by the Admin service definition
+     *
+     * @param array $defaultArguments
+     * @param Definition $definition
+     * @param Definition|null $parentDefinition
+     */
+    public function replaceDefaultArguments(array $defaultArguments, Definition $definition, Definition $parentDefinition = null)
```

Comment:
Ok, I will replace but other methods (applyXXX) in the compiler pass were public so that I was confused about this.